jormungandr-integration-tests/src/common/jcli_wrapper/jcli_transaction_wrapper/jcli_transaction_commands.rs

```diff
@@ -86,6 +86,30 @@ impl TransactionCommands {
         command
     }
 
+    pub fn get_finalize_with_fee_command(
```

Comment:
Many parameters; I'd rather use `LinearFees` if applicable, or invent a parameter structure with public named fields to construct at the call sites.

Reply:
In a parameter structure, the path field is more practical as `PathBuf` to avoid dealing with lifetimes (unless you love lifetime parameters), but values assigned to it can be easily converted from various types with `.into()`.